recipes/tts/all/conanfile.py

```diff
@@ -0,0 +1,48 @@
+import os
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+class TTSConan(ConanFile):
+    name = "tts"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://jfalcou.github.io/tts/"
+    description = "C++ 20 unit tests library focusing on numerical testing and extensibility..."
+    topics = ("c++20", "test", "computing")
+    settings = "compiler"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "10.2",
+            "Visual Studio": "19.27",
```

Comment:
This might be a valid MSVC version number, but it isn't a valid VS version number. It probably should be 16 for Visual Studio 2019?

Reply:
I got tricked by the output of cl.exe then. 16.27 then